src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocBlockTagLocationCheck.java

```diff
@@ -72,6 +72,9 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <p>
  * To configure the default check:
```

Comment:
`To configure the default check:` let use it as one more marker

Reply:
Done. Here is the metadata gen PR:
https://github.com/checkstyle/metadata-gen/pull/46